docs/en/revocation-lists.rst

```diff
@@ -1,57 +1,520 @@
 .. include:: ../common/common_definitions.rst
 
-.. _revocation-lists.rst:
+.. _sec_revocation_intro:
 
-revocation-lists.rst
-+++++++++++++++++++++++++++
+Credential Lifecycle
+++++++++++++++++++++
 
-[What is it]
-    
-[What it is usefull for]
+The value of a Digital Credential is conditional on its validity. A Credential that has been revoked, due to legal requirements, inaccuracy or compromise, is valueless and potentially harmful. 
+For these reasons a robust mechanism for managing the life-cycle and the revocation of a Digital Credential is required.
+
+This section outlines the key technical requirements and processes related to the revocation of Digital Credentials. 
+Furthermore, it provides the technical details that the Verifiers MUST implement to verify, in a secure and reliable manner, the validity of a Digital Credential during the presentation phase.
+
+The verification of the validity of a Digital Credential is based on the `[OAuth Status Attestation draft 00] <https://datatracker.ietf.org/doc/draft-demarco-oauth-status-attestations/00/>`_ specification. 
```

Comment:
```suggestion
The verification of the validity of a Digital Credential is based on the `[OAuth Status Attestation draft 01] <https://datatracker.ietf.org/doc/draft-demarco-oauth-status-attestations/00/>`_ specification. 
```

Reply:
since in 01 we have the credential_pop

Reply:
yes, I changed also the url to point to 01